### PR TITLE
[in_app_purchase] Bump dependencies on json_serializable, build_runner

### PR DESCRIPTION
**Pull request by kevmoo**
Regenerate output
Also put in a pointer to the json_serializable issue related to creating
stand-alone enum converters

### CODE REVIEW

**Comment by cyanglaz** [comment]
LGTM, we need to do a patch version bump, and add a new entry to the CHANGELOG.

**Comment by kevmoo** [comment]
@cyanglaz – there is zero runtime affect here, though.

Do you always release every tweaks to the code?

**Comment by cyanglaz** [comment]
@kevmoo we usually release what makes sense. If it is ok to not release this version, you can add a NEXT entry in the CHANGELOG, so it will be included in the next release. See https://github.com/flutter/plugins/blob/af2896b199ecc3cc5e1ed3000b009f65fc05c9cd/packages/wifi_info_flutter/wifi_info_flutter/CHANGELOG.md#next

**Comment by kevmoo** [comment]
@cyanglaz – PTAL

File: packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Internal changes.
```
Comment (cyanglaz) [inline packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md:3]:
```suggestion
* Update dev_dependency `build_runner` to ^2.0.0 and `json_serializable` to ^5.0.2.
```

File: packages/in_app_purchase/in_app_purchase_ios/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Internal changes.
```
Comment (cyanglaz) [inline packages/in_app_purchase/in_app_purchase_ios/CHANGELOG.md:3]:
```suggestion
* Update dev_dependency `build_runner` to ^2.0.0 and `json_serializable` to ^5.0.2.
```

**Comment by kevmoo** [comment]
@cyanglaz – PTAL

**Review by cyanglaz — APPROVED** [review]
LGTM

**Comment by kevmoo** [comment]
Ooops! Just saw the label, @cyanglaz – did I break something?

**Comment by cyanglaz** [comment]
It should be fine, the label is just for auto-merge :) You can still do it manually after ensuring the ci passes.

**Pull request merged**